### DISCUSSION TITLE
Welcome to DWMBlurGlass Discussions!

### DISCUSSION THREAD

**Discussion by Maplespe** [Maplespe/DWMBlurGlass#157] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by brian080495** [comment]
<img width="715" height="1077" alt="螢幕擷取畫面 2025-09-30 014108" src="https://github.com/user-attachments/assets/61c5ae59-907f-4373-ab29-720ae5b622c6" />
我不知道怎麼安裝，我不熟悉用電腦
